### PR TITLE
Feature/dynamic ac layer

### PR DESCRIPTION
**Pull request by tomermoshe**

### CODE REVIEW

**Comment by tomermoshe** [comment]
ALPHA - have a look.

**Comment by dotansimha** [comment]
Yeah

**Comment by dotansimha** [comment]
image

**Comment by eitanfr** [comment]
fix CI please

File: demo/utils/services/dataProvider/sim-tracks-data-provider.ts [hunk 1]
```diff
@@ -7,43 +7,91 @@ import { AcNotification } from '../../../../src/angular-cesium/models/ac-notific
...
+    entity.array = this.getArrayBySize(entity, Math.round(Math.random() * 3));
+    entity.position = Cesium.Cartesian3.fromDegrees(entity.position.long, entity.position.lat, entity.alt);
+    entity.futurePosition =
+      Cesium.Cartesian3.fromDegrees(entity.futurePosition.long, entity.futurePosition.lat, entity.futurePosition.altitude);
+    const hpr = new Cesium.HeadingPitchRoll(fixedHeading, pitch, roll);
+    entity.orientation = Cesium.Transforms.headingPitchRollQuaternion(entity.position, hpr);
+    return entity;
+  }
+
+  getArrayBySize(entity, size) {
```
Comment (eitanfr) [inline demo/utils/services/dataProvider/sim-tracks-data-provider.ts]: we get the innter arrray for data ffrom the server?

Comment (tomermoshe) [reply demo/utils/services/dataProvider/sim-tracks-data-provider.ts]: just some mock data

**Pull request merged**